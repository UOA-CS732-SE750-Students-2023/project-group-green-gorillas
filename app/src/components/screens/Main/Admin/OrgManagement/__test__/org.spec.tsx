


jest.mock('../../../../../../api/request', () => ({
    patch: jest.fn().mockResolvedValue(undefined)
}))

jest.mock("../../../../../../hooks/useCurrentUser", () => ({
    useCurrentUser: jest.fn().mockReturnValue({
        user: {
            firstName: "",
            lastName: "",
            organisation: {
                name: "Company Name Test",
                id: "001",
                createAt: "2023-05-09T22:02:13.881+00:00",
                updatedAt: "2023-05-11T12:09:10.020+00:00"
            },
        },
    })})
);

import {render, screen} from '@testing-library/react'
import React from "react";
import Org from "../org";


describe('OrgManagement Profile', ()=>{
    it('Display company info', () => {

        const { container } = render(<Org />);

        expect(container).toMatchSnapshot();
    })
})