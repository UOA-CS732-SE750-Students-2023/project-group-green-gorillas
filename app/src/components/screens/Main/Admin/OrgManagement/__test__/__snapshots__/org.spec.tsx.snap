// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrgManagement Profile Display company info 1`] = `
<div>
  <div
    style="height: 400px; width: 100%;"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-19d5vt3-MuiContainer-root"
    >
      <div
        class="MuiBox-root css-wkpw2c"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation5 css-1rvgjkj-MuiPaper-root"
        >
          <div
            class="MuiGrid-root css-dms9wh-MuiGrid-root"
          >
            <h1
              class="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
            >
              Company Info
            </h1>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container css-1vam7s3-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-9omom0-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                Company Name
              </p>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-1vm8tgf-MuiGrid-root"
            >
              <div
                class="MuiFormControl-root MuiFormControl-marginNormal MuiTextField-root css-1rv3ei8-MuiFormControl-root-MuiTextField-root"
              >
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input MuiOutlinedInput-input css-1t8l2tu-MuiInputBase-input-MuiOutlinedInput-input"
                    id=":r0:"
                    required=""
                    type="text"
                    value="Company Name Test"
                  />
                  <fieldset
                    aria-hidden="true"
                    class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="css-ihdtdm"
                    >
                      <span
                        class="notranslate"
                      >
                        â€‹
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-1vm8tgf-MuiGrid-root"
            >
              <button
                aria-label="Disabled elevation buttons"
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-disableElevation css-4kgyf8-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Update
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container css-1wf4x42-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-9omom0-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                Create Time
              </p>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-dweh7w-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                NaN/NaN/NaN
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container css-1wf4x42-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-1cvd8ee-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                Update Time
              </p>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-dweh7w-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                11/05/2023
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container css-1wf4x42-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-6 MuiGrid-grid-sm-4 MuiGrid-grid-md-3 css-1cvd8ee-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                Active State
              </p>
            </div>
            <div
              class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1tnjhf1-MuiGrid-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              >
                False
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
